src/circuits/joinsplit.tcc

```diff
@@ -47,13 +47,19 @@ class joinsplit_gadget : libsnark::gadget<FieldT> {
 
         // ---- Auxiliary inputs (private) ---- //
         libsnark::pb_variable_array<FieldT> zk_total_uint64; // Total amount transfered in the transaction
-        std::array<std::shared_ptr<input_note_gadget<FieldT, HashT, HashTreeT> >, NumInputs> input_notes; // Input note gadgets
-        std::array<std::shared_ptr<output_note_gadget<FieldT, HashT> >, NumOutputs> output_notes; // Output note gadgets
-        std::array<std::shared_ptr<PRF_pk_gadget<FieldT, HashT>>, NumInputs> h_is_gadgets; // Input note gadgets
-        std::array<std::shared_ptr<PRF_rho_gadget<FieldT, HashT> >, NumOutputs> rho_is_gadgets; // Output note gadgets
-        std::shared_ptr<libsnark::digest_variable<FieldT> > phi; // random seed for uniqueness of the new rho
         std::array<std::shared_ptr<libsnark::digest_variable<FieldT> >, NumInputs> a_sks; // List of all spending keys
         std::array<std::shared_ptr<libsnark::digest_variable<FieldT> >, NumOutputs> rho_is; // List of all output rhos
+        std::shared_ptr<libsnark::digest_variable<FieldT> > phi; // random seed for uniqueness of the new rho
+
+
+        // Input note gadgets
+        std::array<std::shared_ptr<input_note_gadget<FieldT, HashT, HashTreeT> >, NumInputs> input_notes; // Input note gadgets
+        std::array<std::shared_ptr<PRF_pk_gadget<FieldT, HashT>>, NumInputs> h_i_gadgets; // Input note gadgets
+
+        // Output note gadgets
+        std::array<std::shared_ptr<PRF_rho_gadget<FieldT, HashT> >, NumOutputs> rho_i_gadgets; // Output note gadgets
```

Comment:
Let's keep one comment:
```c++
        // Output note gadgets
        std::array<std::shared_ptr<PRF_rho_gadget<FieldT, HashT> >, NumOutputs> rho_i_gadgets; // Output note gadgets
```

Should become:
```c++
        // Output note gadgets
        std::array<std::shared_ptr<PRF_rho_gadget<FieldT, HashT> >, NumOutputs> rho_i_gadgets; 
```
ie: I deleted the comment at the end of the line, the comment above the line already gives the necessary information.

Reply:
Fixed